react-native-libraries.json

```
@@ -9879,7 +9879,14 @@
   {
     "githubUrl": "https://github.com/AlirezaHadjar/expo-drag-drop-content-view",
     "examples": ["https://github.com/AlirezaHadjar/expo-drag-drop-content-view/tree/main/example"],
-    "ios": true
+    "images": ["https://raw.githubusercontent.com/AlirezaHadjar/expo-drag-drop-content-view/main/.github/screenshots/ios.gif", 
+      "https://raw.githubusercontent.com/AlirezaHadjar/expo-drag-drop-content-view/main/.github/screenshots/Android.gif", 
+      "https://raw.githubusercontent.com/AlirezaHadjar/expo-drag-drop-content-view/main/.github/screenshots/web.gif"],
+    "npmPkg": "expo-drag-drop-content-view",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "nameOverride": "Native Drag Drop Feature"
```

Hello @AlirezaHadjar, thanks for the updates! 👍 

We can skip `npmPkg` since it matches the repository name, and `nameOverride` is basically for handling edge monorepo cases, and we prefer to use npm package name for title, unless it's necessary to do otherwise.

```suggestion
    "images": [
      "https://raw.githubusercontent.com/AlirezaHadjar/expo-drag-drop-content-view/main/.github/screenshots/ios.gif", 
      "https://raw.githubusercontent.com/AlirezaHadjar/expo-drag-drop-content-view/main/.github/screenshots/Android.gif", 
      "https://raw.githubusercontent.com/AlirezaHadjar/expo-drag-drop-content-view/main/.github/screenshots/web.gif"
    ],
    "ios": true,
    "android": true,
    "web": true
```

---

Hi @Simek , thank you for reviewing the PR. 
Got it. I'll update it right away.